tests/functional/tests/management/test_disable_cleaner.py

```diff
@@ -4,9 +4,17 @@
 #
 
 import pytest
+from pyocf.types.volume import RamVolume
+from pyocf.types.cache import Cache, CacheMetadataSegment
+from pyocf.types.core import Core
+from pyocf.types.volume_core import CoreVolume
+from pyocf.types.data import Data
+from pyocf.types.io import IoDir
```

Comment:
`Data` and `IoDir` is not needed anymore, `CoreVolume` won't be if you remove them